client/lib/titan/get-titan-urls.js

```diff
@@ -12,27 +12,32 @@ const TITAN_APPS = {
  *
  * @param {string?} email - The email address of the Titan account. Used for autofill on Titan's login page.
  * @param {string?} app - Can be one of the `TITAN_APPS` - `email`, `calendar` or `contacts`
+ * @param {boolean?} clearPreviousSessions - Whether to clear previously logged in sessions.
  * @returns The URL with app and prefilled `email_account` as query parameter
  */
-function getTitanUrl( email, app = TITAN_APPS.EMAIL ) {
+function getTitanUrl( email, app = TITAN_APPS.EMAIL, clearPreviousSessions = false ) {
 	const titanBaseUrl = 'https://wp.titan.email';
 	const titanAppUrl = new URL( `${ titanBaseUrl }/${ app }` );
 
 	if ( email?.includes( '@' ) ) {
 		titanAppUrl.searchParams.append( 'email_account', email );
 	}
 
+	if ( clearPreviousSessions ) {
+		titanAppUrl.searchParams.append( 'clearSession', 'true' );
+	}
+
 	return titanAppUrl.href;
 }
 
-export function getTitanCalendarlUrl( email ) {
-	return getTitanUrl( email, TITAN_APPS.CALENDAR );
+export function getTitanCalendarlUrl( email, clearPreviousSessions = false ) {
+	return getTitanUrl( email, TITAN_APPS.CALENDAR, clearPreviousSessions );
```

Comment:
I don't think we should update that method since we don't use that new parameter when we call it yet. We can update it when we really need it.